test/plugins/bind-dictionary.js

```diff
@@ -1,20 +1,9 @@
-/* eslint-env node, browser, mocha */
-/* eslint no-unused-expressions:0 */
-'use strict'
-/* global
-    Event
-*/
-require('es5-shim/es5-shim')
-require('es5-shim/es5-sham')
+/* eslint-env browser, mocha */
 var assert = require('proclaim')
 var sinon = require('sinon')
 var Combokeys = require('../..')
 var KeyEvent = require('.././lib/key-event')
 
-afterEach(function () {
-  Combokeys.reset()
-})
-
```

Comment:
This change appears in other files, as well. It is a relic. It should have been removed when instantiation was added.